### DISCUSSION TITLE
繁简混合输入

### DISCUSSION THREAD

**Discussion by tomatl45** [rime/home#1063] [Q&A]
你好，

我想实现这样一种功能：在输入文字的时候，可以不必按 Ctrl + ` 切换繁简，直接同时输入包含繁体和简体的词语或短句。

一个可能的方法：将简体和繁体视为不同的字符，共存于一个输入方案内，分别记忆词频、自造词等。

请问这种功能该通过哪些操作实现？

谢谢。

**Comment by LEOYoon-Tsaw** [comment]
製作一份兼含簡繁之碼表

**Comment by ShikiSuen** [comment]
窃以为这个产品需求本来就非常麻烦。

LEO 的提案无法统一简体中文语料与繁体中文语料各自的权重差异。
哪怕是威注音输入法，简体中文与繁体中文各自独立为一个输入法副本（就像 macOS 内建的日语输入法那样），也得来回切换输入法。

不是说提了产品需求就一定能实现的，还得看值不值。每个专案的情况都不同，且对这「值不值」的看法也是因人而异。

**Reply by LEOYoon-Tsaw** [reply]
只要提問者有毅力，修正權重又不是多難的事情，花點時間而已
